index.js

```diff
@@ -36,6 +36,14 @@ function GithubActionsReporter(runner, options) {
 
             core.endGroup();
         }
+        core.summary.addRaw(`:white_check_mark: ${this.stats.passes || 0} passing (${milliseconds(this.stats.duration)})`, true);
+        if (this.stats.pending) {
+            core.summary.addRaw(`:pause_button: ${this.stats.pending} pending`, true);
+        }
+        if (this.stats.failures) {
+            core.summary.addRaw(`:x: ${this.stats.failures} failing`, true);
+        }
+        core.summary.write();
```

Comment:
I've tried to keep this as close to the base epilogue as possible:

https://github.com/mochajs/mocha/blob/37deed262d4bc0788d32c66636495d10038ad398/lib/reporters/base.js#L374-L396

Feel free to adjust or remove the [emojis](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md).